app/src/pages/inside/dashboardPage/dashboardList/dashboardTable/dashboardTable.jsx

```diff
@@ -59,6 +60,8 @@ const messages = defineMessages({
 @connect((state) => ({
   projectId: activeProjectSelector(state),
```

Comment:
Do we need it?

Reply:
We use it in UI, to display userfrendly names.